### ISSUE TITLE
GDLauncher crashes when you scroll to the bottom of a modpack list

### ISSUE THREAD

**Issue by z810d** [killpowa/Twickt-Launcher#1589]
- [X] I searched the issue tracker for other issues covering my case
- [X] This is a Launcher issue. (Usually if you can reproduce it with a fresh instance)

**Describe the bug**
When you scroll to the bottom of the FTB modpack list, GDLauncher crashes and asks you to restart.

**To Reproduce**
1. Click 'Add new Instance'
2. Click on 'FTB'
3. Scroll down to the bottom at whatever speed you want
4. Encounter error

**Expected behavior**
Shouldn't crash when you get to the bottom of the list of modpacks.

**Possible solutions**
_Some ideas of how you would solve this issue, if possible_

**Screenshots**
_If applicable, add screenshots to help explain your problem._

**Operating System:**
- OS: Windows 10
- Launcher Version: v1.1.30
- Java Version: 17
- Minecraft Version: N/A
- Mod Loader: N/A

**Additional context**
This is a low priority issue, and it can be worked around in the interim. Most users would just search for the specific pack that they want instead of browsing anyways.

**Comment by Eskaan** [comment]
Yeah, FTB most likely won't work any longer anyways. They have changed their API I think which might be the reason to crashes too.
